tests/dags/providers/test_provider_dag_factory.py

```diff
@@ -0,0 +1,48 @@
+from providers import provider_dag_factory
+
```

Comment:
Appreciate this, I had _just_ made a note this morning to move this file when I went looking for it under this directory and couldn't find it! Nice ✨